x/mongo/driver/topology/server_test.go

```diff
@@ -1262,20 +1263,25 @@ func includesClientMetadata(t *testing.T, wm []byte) bool {
 // for Server.ProcessError. This type should not be used for other tests
 // because it does not implement all of the functions of the interface.
 type processErrorTestConn struct {
+	mnet.WireMessageReadWriteCloser
+	mnet.Describer
 	// Embed a driver.Connection to quickly implement the interface without
 	// implementing all methods.
-	driver.Connection
 	description description.Server
 	stale       bool
 }
 
-func newProcessErrorTestConn(wireVersion *description.VersionRange, stale bool) *processErrorTestConn {
-	return &processErrorTestConn{
+func newProcessErrorTestConn(t *testing.T, wireVersion *description.VersionRange, stale bool) *mnet.Connection {
+	t.Helper()
```

Comment:
Is there a reason to call `t.Helper` here? Typically the purpose is to allow failed test assertions to point to a more relevant line number, but `t` is not used to make any assertions in this function.